### DISCUSSION TITLE
Add Finer Grain Edit/View Permissions for Users

### DISCUSSION THREAD

**Discussion by a-blind-programmer** [gramps-project/gramps-web-api#536] [Ideas]
### Idea

Limit users to viewing/editing a list of specific people with the option to allow viewing/editing of that person's:

1. Descendents
2. Ancestors
3. Specified Degree of Separation

**Comment by DavidMStraub** [comment]
That would be nice, but unfortunately it is very difficult to achieve given the way Gramps interacts with the database. In any case, it's a backend question, so I'll move it to a discussion in the Web API repo.
